src/Fakta.IntegrationTests/Audit.fs

```diff
@@ -0,0 +1,52 @@
+﻿module Fakta.IntegrationTests.Audit
+
+open System
+open System.Net
+open Chiron
+open Chiron.Operators
+open Fuchu
+open NodaTime
+open Fakta
+open Fakta.Logging
+open Fakta.Vault
+
+let filePath = @"C:\audit.log"
```

Comment:
It's not ideal, but rather unexpectedly it does work. It just writes a file called "c:\audit.log" in the working directory.